caddyhttp/httpserver/replacer.go

```diff
@@ -375,6 +376,26 @@ func (r *replacer) getSubstitution(key string) string {
 		}
 		elapsedDuration := time.Since(r.responseRecorder.start)
 		return strconv.FormatInt(convertToMilliseconds(elapsedDuration), 10)
+	case "{ssl_protocol}":
```

Comment:
Change to `{tls_protocol}` or, perhaps, `{tls_version}`. Open for discussion!

Reply:
To avoid any arguments, alias; i.e. how about using either ```{tls_...}``` or ```{ssl_...}``` (or is an axtra case statement too much?) Discuss!

Reply:
This would lead to unnecessary complication...

Reply:
Just use `tls_` :wink: